store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/integration/AccumuloStoreITs.java

```diff
@@ -28,7 +28,7 @@
 import static uk.gov.gchq.gaffer.integration.junit.extensions.IntegrationTestSuiteExtension.INIT_CLASS;
 
 @ConfigurationParameter(key = INIT_CLASS, value = "uk.gov.gchq.gaffer.accumulostore.integration.AccumuloStoreITs")
-@SelectClasses(AddElementsFromHdfsLoaderIT.class) // Extra test to add
+@SelectClasses(AddElementsFromHdfsLoaderIT.class)
```

Comment:
I think this comment was actually okay

Reply:
I added this originally when refactoring this class. No changes needed to be made to this file as part of this PR.